### ISSUE TITLE
Notifications black background not applied

### ISSUE THREAD

**Issue by g0to** [InFlames03/Oreo-Theme#1]
I can't manage to correctly apply changes to the System UI overlay. My notifications background stays white, although the text color changes. This makes some notifications text invisible since font and background colors are the same.
Manual rebooting hasn't worked so far :/   

I have also installed [Navbars Theme Ex](https://play.google.com/store/apps/details?id=n.n.Navbars). I wonder if there are any incompatibilities between the two themes.

### Notifications behaviour
![screenshot_20171220-112139](https://user-images.githubusercontent.com/4366508/34206093-ed75524a-e57c-11e7-993e-e5a71463b66c.png)

### Substratum configuration
![screenshot_20171220-111306](https://user-images.githubusercontent.com/4366508/34206118-0e69db06-e57d-11e7-9eb8-262f8aa08c34.png)
![screenshot_20171220-111314](https://user-images.githubusercontent.com/4366508/34206136-1cc92620-e57d-11e7-8757-d6510faabcde.png)

### Android version 
![screenshot_20171220-112744](https://user-images.githubusercontent.com/4366508/34205814-cf869bd2-e57b-11e7-8696-02c571f3207e.png)

**Comment by InFlames03** [comment]
The issue is your SystemUI overlay being in a "Disabled" state. The colour being orange is the indicator for that, so please manually enable it and then perform a system reboot for it to take full effect (if using Andromeda, if not using Andromeda, restart SystemUI).

- Sai

**Comment by g0to** [comment]
I've tried that already and the overlay status goes always back to disabled. Sorry for not mentioning that earlier.

**Comment by InFlames03** [comment]
Hmm, that shouldn't happen. Only thing I can suggest right now is to uninstall and reinstall all the themes. It may or may not be helpful if you send me your Substratum app version number along with the backend/theming method you use (Andromeda or Root).

**Comment by g0to** [comment]
After uninstalling all my themes and installing them again progressively, I've discovered that there's an incompatibility between this theme and [Navbars Theme Ex](https://play.google.com/store/apps/details?id=n.n.Navbars). When the latter is enabled, black notifications start acting funny.

I'm using Andromeda (version 18) with Substratum (version 914).

Let me know if you need more info.

**Comment by InFlames03** [comment]
I can't do anything about it if it's the fault of Navbars Theme Ex. I'll be bringing back my navbar overlays in the next update however, so feel free to get rid of Navbars Theme Ex once I do so.

Also, first of all, a navbar overlay should never mess with notifications. Either something wrong with the application of the theme itself, or the dev of Navbars Theme Ex needs to rework their implementation. I will look more into this later and find the root cause.

- Sai

**Comment by g0to** [comment]
I've tried two more navbar overlays, [N Navbar - Substratum](https://play.google.com/store/apps/details?id=com.varefaz.NNavbar&rdid=com.varefaz.NNavbar)
and [[Substratum] Navigate Theme](https://play.google.com/store/apps/details?id=com.rc.navigate), and the problem persists. When they're enabled, your theme's System UI option is automatically disabled, making the notifications look weird.

Do you know when the new version will be available? The option to make the navbar buttons transparent is exactly what I'm looking for. 🙂

Thanks.

**Comment by InFlames03** [comment]
Toggle the "Disable other overlays" option and re-apply system UI overlays

**Comment by g0to** [comment]
> Toggle the "Disable other overlays" option and re-apply system UI overlays

I can't find that option in Substratum 915. How did you get it working? If it does what I think, I'll lose the navbars overlay again.

Any updates about when the new version would be available?

Thanks.

**Comment by InFlames03** [comment]
![This is where you can find the option](https://i.imgur.com/PhSImwW.png)

**Comment by g0to** [comment]
Thanks for the screenshot. 

Unfortunately, that didn't help either. Enabling this theme's black notifications disables the System UI Navigation icons overlay on other themes and vice versa.

**Comment by InFlames03** [comment]
Yeah, I've tried hard to reproduce this issue but I just can't seem to do it. I used Gaia's navbar overlay along with mine simultaneously on both 8.0 and 8.1 and they worked like a charm in unison.
